### ISSUE TITLE
[Bug]: Binance Wallet (injected) broken as of 2.13.1

### ISSUE THREAD

**Issue by ReflectiveChimp** [blocknative/web3-onboard#1169]
### Current Behavior

> Cannot add property isUnlocked, object is not extensible

![image](https://user-images.githubusercontent.com/55021052/180987625-20a0a61b-c255-4e30-9f6c-9125bd5d0dce.png)

### Expected Behavior

Binance Wallet should connect and function.

### Steps To Reproduce

1. Install latest (2.13.1) Binance Wallet extension: https://chrome.google.com/webstore/detail/binance-wallet/fhbohimaelbohpjbbldcngcnapndodjp and create account/address.
2. Go to https://reactdemo.blocknative.com/
3. Press Select a Wallet
4. Select Binance Smart Wallet
5. See error `Cannot add property isUnlocked, object is not extensible`

### What package is effected by this issue?

@web3-onboard/injected

### Is this a build or a runtime issue?

Runtime

### Package Version

2.5.0

### Node Version

16.16.0

### What browsers are you seeing the problem on?

Chrome

### Anything else?

Works with previous versions of the extension: v2.13.0 and down.

Tested locally with 
```
"@web3-onboard/core": "^2.5.0",
"@web3-onboard/injected-wallets": "^2.0.14",
```

Seems something changed in the extension that no longer allows writing to window.BinanceChain here:
https://github.com/blocknative/web3-onboard/blob/fb81fbb38c5629bde0fac681e86bfe382092bda4/packages/injected/src/wallets.ts#L71

### Sanity Check

- [X] If this is a build issue, I have included my build config. If this is a runtime issue, I have included reproduction steps and/or a [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example).

**Comment by Adamj1232** [comment]
@ReflectiveChimp lookin into this now. Will update here on status changes

**Comment by Adamj1232** [comment]
@ReflectiveChimp pr to fix this issue - #1174

**Comment by Adamj1232** [comment]
@ReflectiveChimp A fix is merged into staging and available for testing with the `2.0.15-alpha.2` version of the injected package. A production release will happen in the next 24hrs

**Comment by ReflectiveChimp** [comment]
@Adamj1232 Thank you for quick turn around, can confirm the fix works.
